src/components/App/App.css

```diff
@@ -20,7 +20,7 @@ body {
 
 .chatContainer {
   flex-grow: 1;
-  overflow: scroll;
+  overflow: auto;
```

Comment:
what was this change fixing?

Reply:
There were scrollbars on the chat even though they are not needed when the chat is too short.